packages/manager/apps/pci-public-ip/src/interface/index.ts

```diff
@@ -19,3 +19,32 @@ export interface FloatingIP {
   region: string;
   status: string;
 }
+
+export interface FailoverIP {
+  associatedEntity: Instance | null;
+  block: string;
+  continentCode: 'EU';
```

Comment:
should be type string no?

Reply:
you can check the ResponseClass visible in api ovh com if you have any doubt about return types